### PR TITLE
TS Flexible key tests

### PR DESCRIPTION
**Pull request by andytill**

### CODE REVIEW

File: tests/ts_cluster_keys.erl [hunk 1]
```diff
@@ -0,0 +1,169 @@
+-module(ts_cluster_keys).
```
Comment (javajolt) [inline tests/ts_cluster_keys.erl]: Sorry to be nit picky here, @andytill, but how about `ts_cluster_flexible_keys`? Just `keys` seems a wee bit vague.

**Comment by javajolt** [comment]
Guessing that https://github.com/basho/riak_kv/pull/1333 is required for this PR, too.

File: tests/ts_cluster_keys.erl (hunk 1 above)
Comment (andytill) [inline tests/ts_cluster_keys.erl]: The key changes seem flexible now but in the feature we will think of them just as keys.

**Comment by andytill** [comment]
Closing, going to open a new PR to 1.3

**Pull request closed without merging**